src/components/Form.js

```diff
@@ -18,16 +18,23 @@ const Form = (props) => {
 
     return (
       <div>
-          <form onSubmit={handleSubmit}>
-              <input 
+          {/* For Job Title Search */}
+        <form onSubmit={handleSubmit}>
+              {/* <input 
```

Comment:
I think it would be good to remove the commented out code for now and we can add it back as another component later